expr/expr.go

```diff
@@ -3897,7 +3906,25 @@ func (w *windowed) Stdev() float64 {
 }
 
 func (w *windowed) Mean() float64 { return w.sum / float64(w.Len()) }
+func (w *windowed) Max() float64 {
```

Comment:
The original graphite-web does a naive approach like I proposed in this PR, but agreed, this could be much faster.

I added a note. Selfishly, my use case is just 5 data points, so I'm OK with the slow approach :)